### PR TITLE
Hosted video

### PR DESCRIPTION
**Pull request by Calanthe**
## What does this change?

This PR is the continuation of the https://github.com/guardian/frontend/pull/12832. The hosted page will include at least one video with a specific behaviour (fade in and out of control buttons) and styling (bigger control buttons, different icon's colour based on the brand). I created a new, much simplified video module, since we don't need now any additional features like playlists or video pre-rolls. 
## What is the value of this and can you measure success?

We will see if people will watch the video.
## Does this affect other platforms - Amp, Apps, etc?

At some point it will be.
## Screenshots

hosted_video
## Request for comment

@uplne

### CODE REVIEW

File: static/src/javascripts/bootstraps/commercial.js [hunk 1]
```diff
@@ -36,7 +38,8 @@ define([
         ['cm-articleBodyAdverts', articleBodyAdverts.init],
         ['cm-sliceAdverts', sliceAdverts.init],
         ['cm-frontCommercialComponents', frontCommercialComponents.init],
-        ['cm-commercialAudit', commercialAudit.init]
+        ['cm-commercialAudit', commercialAudit.init],
+        ['cm-hostedVideo', hostedVideo.init]
```
Comment (uplne) [inline static/src/javascripts/bootstraps/commercial.js]: Task for the future next generation of hosted content is to include only hosted scripts as we don't need most of the commercial stuff there.

**Comment by uplne** [comment]
Magic 👍

File: static/src/javascripts/projects/common/modules/commercial/hosted-video.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            initLoadingSpinner(player);
+            upgradeVideoPlayerAccessibility(player);
+            supportedBrowsers(player);
+        });
+    }
+
+    return {
+        init: init
+    };
+});
```
Comment (jamesgorrie) [inline static/src/javascripts/projects/common/modules/commercial/hosted-video.js]: 🎆 for not moving this into the messy `media/main` Thanks!

File: static/src/stylesheets/module/commercial/glabs/_hosted.scss [hunk 3]
```diff
@@ -84,34 +94,112 @@ $hosted-stripe: 28px;
...
+    .vjs-volume-menu-button,
+    .vjs-progress-control {
+        opacity: 0;
+    }
+
+    &.vjs-paused {
+        .vjs-fullscreen-control,
+        .vjs-volume-menu-button,
+        .vjs-progress-control {
+            @include hosted-fade-out;
```
Comment (jamesgorrie) [inline static/src/stylesheets/module/commercial/glabs/_hosted.scss]: Perhaps I am mistaken - but once paused, would this make it impossible to play again?

Comment (jamesgorrie) [inline static/src/stylesheets/module/commercial/glabs/_hosted.scss]: Or is the play button in the `hosted__meta `?

**Comment by jamesgorrie** [comment]
Super!
I wonder if we could steal this for fronts? @piuccio @akash1810

File: static/src/stylesheets/module/commercial/glabs/_hosted.scss (hunk 3 above)
Comment (Calanthe) [inline static/src/stylesheets/module/commercial/glabs/_hosted.scss]: The play button is in the `vjs-control-bar` which is always visible (only the background colour changes).

**Pull request merged**